docs/how-to/choose-configure-update-database.md

```diff
@@ -0,0 +1,64 @@
+---
+id: choose-configure-update-database
+title: How To Choose, Configure, and Update Your Database
+description: Learn how to choose, configure, and update your Amplication database using plugins.
+sidebar_label: How To Choose, Configure, and Update Your Database
+slug: /plugins/guides/database
+---
+
+# How To Choose, Configure, and Update Your Database
+
+One important part of setting up your Amplication service is choosing and configuring your database. Your database is a _plugin_ in Amplication, so can operate it like other plugins.
+
+This guide will walk you through the process of selecting, configuring, and updating your database in Amplication.
+
+:::note
+Amplication's [Import Prisma Schema](/how-to/import-prisma-schema/) feature lets you use your existing databases for a head start on development.
+:::
+
+## Choosing Your Database
+
+Amplication supports multiple database options, with PostgreSQL being the default choice. Here are the steps to choose your database:
+
+1. When [creating a new service](/first-service/) in Amplication, you'll be required to select a database.
```

Comment:
This is relevant only in case creating a service using the wizard. When creating a service using Jovu- the user can ask for a different database than the default.
Let's make it clearer